src/main/java/org/fogbowcloud/arrebol/api/http/dataaccessobject/JobDAO.java

```diff
@@ -0,0 +1,33 @@
+package org.fogbowcloud.arrebol.api.http.dataaccessobject;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.arrebol.core.models.job.Job;
+import org.fogbowcloud.arrebol.core.repositories.JobRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class JobDAO {
+
+    private final Logger LOGGER = Logger.getLogger(JobDAO.class);
+
+    @Autowired
+    private JobRepository jobRepository;
+
+    public Job getJobById(String id){
+        LOGGER.debug("Acessing database to getting a job");
```

Comment:
In most of the cases, a log line without a variable smells bad. This is the case of this method. Please, add the id of the job (as well as the other method of this class).